MealsApp/android/app/src/main/kotlin/com/example/flutter_proj5meal/MainActivity.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.flutter_proj5meal
```

Comment:
Update package name.

You will need to use the "Refactor" function, because the directory structure also needs to be updated.
![Screenshot 2021-10-06 at 09 28 05](https://user-images.githubusercontent.com/2096087/136158793-564b010b-7f09-4b20-ab5e-f3c28deb307c.png)

Reply:
@nisrulz I changed the path. but how do I refactor ?